### ISSUE TITLE
[Release 4.1] pme command deprecated

### ISSUE THREAD

**Issue by Victory61** [Patman86/x265-Mod-by-Patman#37]
Good evening
Using the latest release “4.1+54+9-0c07fe364-.Mod-by-Patman.-x64-avx512-msvc1942” with StaxRip, the pme (Parallel Motion Estimation) function has been deprecated.
I wanted to know if disabling this function can create quality problems in coding.
Thank you

![Screenshot (1372)](https://github.com/user-attachments/assets/798bd2e7-b8f8-492f-9072-5312c8865976)

**Comment by DJATOM** [comment]
No, that option actually used for speed. Probably it was replaced with another option or motion estimation got a better logic, I haven't examined those changes yet. So it should be safe to remove.

**Comment by Victory61** [comment]
Thank you for confirmation.
Since I don't know which parameter to activate in place of “pme” I disable it, I assume that this release of x265 automatically “replaces” that function.

**Comment by Patman86** [comment]
Have a look here:

https://bitbucket.org/multicoreware/x265_git/commits/5c2da201444b7f13a6414d970a39e5dfb6452535

**Comment by Victory61** [comment]
Thank you for the explanation.
